orion-ui/src/pages/Deployment.vue

```diff
@@ -76,4 +77,11 @@
   function routeToDeployments(): void {
     router.push(routes.deployments())
   }
+
+  watchEffect(() => {
```

Comment:
as nice as `watchEffect` is I think we should use a `watch` with `{ immediate: true }` here so we can watch just the deployment. `watchEffect` will watch reactive data even in method calls. Meaning if `showToast` ever has reactive data referenced those will trigger the effect. `watch` is just a lot safer.

Reply:
Yup you're correct, updated 👍🏻